src/ios/CDVInAppBrowser.m

```diff
@@ -783,6 +793,16 @@ - (void)showToolBar:(BOOL)show : (NSString *) toolbarPosition
 
 - (void)viewDidLoad
 {
+    CGRect frame = [UIApplication sharedApplication].statusBarFrame;
+    UIToolbar* bgToolbar = [[UIToolbar alloc] initWithFrame:frame];
+    bgToolbar.barStyle = UIBarStyleDefault;
+    bgToolbar.tintColor = [UIColor whiteColor];
+    bgToolbar.barTintColor = [UIColor whiteColor];
+//    [[UIBarItem appearance] setTitleTextAttributes:@{UITextAttributeTextColor : [UIColor whiteColor]}];
+
+
+    [self.view addSubview:bgToolbar];
+    
```

Comment:
What's the purpose of this code?

Reply:
@steinwelberg Can you please create the new merge request to get the essential part in? Clearly it looks like the extra code is causing the delay. It really helps to have the changes in the official branch than to fork and use.